src/cmd/append.go

```diff
@@ -19,6 +19,18 @@ type appendConfig struct {
 var appendCommand = &cobra.Command{
 	Use:   "append <branch>",
 	Short: "Creates a new feature branch as a child of the current branch",
+	Long: `Creates a new feature branch as a child of the current branch.
+
+Syncs the current branch,
+forks a new feature branch with the given name off the current branch,
+makes the new branch a child of the current branch,
+pushes the new feature branch to the remote repository,
+and brings over all uncommitted changes to the new feature branch.
+
+Additionally, when there is a remote upstream,
+the main branch is synced with its upstream counterpart.
```

Comment:
Note: might be good to wait for #934 to merge and then update these accordingly as I removed this part from the markdown docs in that PR.